src/js/utils/cursor.js

```diff
@@ -147,8 +147,13 @@ const Cursor = class Cursor {
 
     const range = document.createRange();
     range.setStart(node, offset);
-    this.selection.addRange(range);
-    this.selection.extend(endNode, endOffset);
+    if (direction === DIRECTION.BACKWARD && !!this.selection.extend) {
```

Comment:
Hm, I don't think so. The `supportsSelectionExtend` actually does a test when the JS is initially executed, and we don't need that here. I'd like to leave them split apart.